doc/pages/doxy_c_api.rst

```diff
@@ -28,15 +72,86 @@ to be reported to NRM.
 .. note::
   For example, if reporting power usage for one socket and one GPU,
   you may want to instantiate two scopes. For the first scope, while looping over corresponding
-  CPU logical indexes, call ``nrm_scope_add()`` with the ``NRM_SCOPE_TYPE_CPU`` type
+  CPU logical indexes, call ``nrm_scope_add()`` with ``NRM_SCOPE_TYPE_CPU``
   and set ``num`` to each index.
 
 .. doxygenfunction:: nrm_scope_length
 
 .. doxygenfunction:: nrm_scope_destroy
 
+.. doxygenfunction:: nrm_scope_dup
+
+.. doxygenfunction:: nrm_scope_cmp
+
 .. doxygenfunction:: nrm_scope_snprintf
 
 .. doxygenfunction:: nrm_scope_threadshared
 
 .. doxygenfunction:: nrm_scope_threadprivate
+
+.. _sensors:
+
+Sensor Configuration
+--------------------
+
+An NRM ``sensor`` corresponds to events to be reported to NRM.
+
+.. doxygenfunction:: nrm_sensor_create
+
+.. doxygenfunction:: nrm_sensor_destroy
+
+Slice Configuration
+-------------------
+
+An NRM ``slice`` names and assigns a uuid to a set of resources.
+
+.. doxygenfunction:: nrm_slice_create
+
+.. doxygenfunction:: nrm_slice_destroy
+
+.. doxygenfunction:: nrm_slice_fprintf
+
+.. _logs:
+
+Logging Interface
+-----------------
+
+.. doxygenfunction:: nrm_log_init
+
+.. doxygenfunction:: nrm_log_setlevel
+
+Log Levels include:
+
+.. literalinclude:: ../../include/nrm.h
+  :lines: 71-76
+  :language: C
+
```

Comment:
use the doxygendefine, that's more reliable, otherwise it needs to be updated everytime we modify the header.